packages/client/src/store/modules/grants.js

```diff
@@ -212,6 +213,43 @@ export default {
       fetchApi.get(`/api/organizations/${rootGetters['users/selectedAgencyId']}/interested-codes`)
         .then((data) => commit('SET_INTERESTED_CODES', data));
     },
+    async getFollowerForGrant({ rootGetters }, { grantId }) {
+      try {
+        return await fetchApi.get(`/api/organizations/${rootGetters['users/selectedAgencyId']}/grants/${grantId}/follow`);
+      } catch (e) {
+        return null;
+      }
+    },
+    async getFollowersForGrant({ rootGetters }, { grantId, limit, paginateFrom }) {
+      const queryParams = serializeQuery({ limit, paginateFrom });
+      try {
+        return await fetchApi.get(`/api/organizations/${rootGetters['users/selectedAgencyId']}/grants/${grantId}/followers${queryParams}`);
+      } catch (e) {
+        return null;
+      }
+    },
```

Comment:
Feel free to disregard if you still think the helper function is convenient and/or you want to avoid dangling `?` characters in URLs, but per my previous comment, this could also be implemented to use `URLSearchParams` directly, e.g.
```js
    async getFollowersForGrant({ rootGetters }, { grantId, limit, paginateFrom }) {
      const queryParams = new URLSearchParams({ limit, paginateFrom });
      try {
        return await fetchApi.get(`/api/organizations/${rootGetters['users/selectedAgencyId']}/grants/${grantId}/followers?${queryParams}`);
      } catch (e) {
        return null;
      }
    },
```